pkgs/tools/wayland/enkei/default.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, glib
+, cairo
+, wayland
+, libGL
+}:
+
+let
+  libdeps = [ wayland.dev libGL.dev glib.dev cairo.dev ];
+in
+
+rustPlatform.buildRustPackage rec {
+  pname = "enkei";
+  version = "v0.9.3";
+
+  src = fetchFromGitHub {
+    owner = "jwuensche";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-COU2JtiJcPRA3Jno0qLEIVgimYBWfn5Pgc1OMImsJtI=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ] ++ libdeps;
+  buildInputs = [
+  ];
+
+  PKG_CONFIG_PATH = lib.strings.makeSearchPath "lib/pkgconfig" libdeps; # why is this necessary to do manually?! (thought it was automagic?)
```

buildRustPackage has strictDeps enabled to lib deps in nativeBuildInputs won't work.
```suggestion
  buildInputs = [ wayland libGL glibc cairo ];
```
this will work

---

@Artturin sorry, completely forgot about this PR...